Tests/Indicators/PivotPointsHighLowTests.cs

```diff
@@ -117,6 +117,95 @@ public void PivotPointPerType(PivotPointType pointType)
+            var pivotPoints = pointsHighLow.GetAllPivotPointsArray();
+            Assert.AreEqual(0, pivotPoints.Length, "With strict checking, equal values should not create pivot points");
+        }
+
+        [Test]
+        public void RelaxedCheckingAllowsTies()
+        {
+            // Test that relaxed checking (strictChecking = false) allows ties
+            var pointsHighLow = new PivotPointsHighLow(2, strictChecking: false);
+
+            // Create data where middle point equals neighbors - should be a pivot with relaxed checking
+            var bars = new[]
+            {
+                new TradeBar(DateTime.UtcNow.AddSeconds(0), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1
+                new TradeBar(DateTime.UtcNow.AddSeconds(1), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+                new TradeBar(DateTime.UtcNow.AddSeconds(2), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (middle - same as neighbors)
+                new TradeBar(DateTime.UtcNow.AddSeconds(3), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+                new TradeBar(DateTime.UtcNow.AddSeconds(4), Symbols.AAPL, 1, 10, 1, 1, 1), // high=10, low=1 (same as middle)
+            };
+
+            foreach (var bar in bars)
+            {
+                pointsHighLow.Update(bar);
+            }
+
+            // With relaxed checking, pivot points should be detected even with equal values
+            var pivotPoints = pointsHighLow.GetAllPivotPointsArray();
+            Assert.IsTrue(pivotPoints.Length > 0, "With relaxed checking, equal values should create pivot points");
+        }
+
+        [Test]
+        public void StrictVsRelaxedComparison()
+        {
+            var strictIndicator = new PivotPointsHighLow(2, strictChecking: true);
+            var relaxedIndicator = new PivotPointsHighLow(2, strictChecking: false);
+
+            // Test data with a clear high pivot point followed by ties
+            var bars = new[]
+            {
+                new TradeBar(DateTime.UtcNow.AddSeconds(0), Symbols.AAPL, 1, 5, 1, 1, 1),   // high=5, low=1
+                new TradeBar(DateTime.UtcNow.AddSeconds(1), Symbols.AAPL, 1, 5, 1, 1, 1),   // high=5, low=1
+                new TradeBar(DateTime.UtcNow.AddSeconds(2), Symbols.AAPL, 1, 10, 1, 1, 1),  // high=10, low=1 (clear high pivot)
+                new TradeBar(DateTime.UtcNow.AddSeconds(3), Symbols.AAPL, 1, 5, 1, 1, 1),   // high=5, low=1
+                new TradeBar(DateTime.UtcNow.AddSeconds(4), Symbols.AAPL, 1, 5, 1, 1, 1),   // high=5, low=1
+                // Now add bars with ties
+                new TradeBar(DateTime.UtcNow.AddSeconds(5), Symbols.AAPL, 1, 8, 1, 1, 1),   // high=8, low=1
+                new TradeBar(DateTime.UtcNow.AddSeconds(6), Symbols.AAPL, 1, 8, 1, 1, 1),   // high=8, low=1 (middle - tied)
+                new TradeBar(DateTime.UtcNow.AddSeconds(7), Symbols.AAPL, 1, 8, 1, 1, 1),   // high=8, low=1
```

Comment:
Minor but ideally instead of `DateTime.UtcNow` should just create new datetime at the top and just use it or even 1 private field `_referenceTime`, no need to use actual time which is more expensive